docs/guidelines/content-guidelines/02-style-and-terminology.md

```diff
@@ -128,41 +130,58 @@ Use short command line arguments whenever possible.
 
 In such a case, explain the context in the document.
 
-## Capitalized terms
-
-This is the list of the Kubernetes resources capitalized in Kyma documentation. However, if these words are not used in relation to Kubernetes resources, do not capitalize them. Use natural language when referring to Kubernetes resources. Do not use CamelCase. 
-
-- Config Map
-- Cron Job
-- Custom Resource Definition
-- Deployment
-- Function
-- Ingress
-- Namespace
-- Node
-- Pod Preset
-- Pod
-- Prow Job
-- Secret
-- Service
-- Service Binding
-- Service Class
-- Service Instance
-
-All Kyma components, such as Application Connector or API Gateway Controller, are capitalized as well. Whenever you point to the outside sources, do some research to check whether the name of the source starts with a capital letter or not.
-
->**NOTE:** Kubernetes itself is also capitalized. Do not write it in lowercase or in the abbreviated version.   
-  ✅ Kubernetes  
-  ⛔️ kubernetes   
-  ⛔️ k8s
+## Capitalization
+
+### Sentence case
+
+Sentence case is the normal style in English: Lowercase for all words, first letter of the first word in a sentence is uppercase. Exception: Uppercase for the first letters of proper nouns and names.
+
+Use sentence case for normal, everyday words in body text. Don't capitalize words just because they are "special" or "important".
+Also, we use use sentence case for headings (more details about [headings](03-formatting.md#headings)).
+
+### Upper case
 
-### Custom resources
+Upper case means the first letter of a word is capitalized.
```

Comment:
This is actually not what Upper case means. If you talk about a letter, upper case means that a letter is --simply -- in upper case (`A`, not `a`). If you talk about a piece of text being written in upper case, it means that THE WHOLE TEXT IS WRITTEN IN UPPER CASE JUST LIKE HERE. 
What you mean is probably **Title Case**, where the first letter of each important word is capitalized ("important" meaning that function words, such as 'of', 'and', etc. are excluded from this and written in lowercase).

See the [Cambridge dictionary](https://dictionary.cambridge.org/dictionary/english/upper-case) for reference.

Reply:
Thank you, I'll fix that!